lib/action.ts

```
@@ -17,9 +17,12 @@ export const getAllProduct = async () => {
     }
 }
 
-export const addProduct = async (values: z.infer<typeof addProductSchema>) => {
+export const addProduct = async (values: z.infer<typeof addProductSchema>, imageUrl: String[]) => {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)